service/src/main/java/org/cbioportal/genome_nexus/service/SelectedAnnotationService.java

```diff
@@ -42,15 +43,15 @@
 public interface SelectedAnnotationService {
 
     public VariantAnnotation getAnnotation(
-            String variant,
-            String isoformOverrideSource,
-            Map<String, String> token,
-            List<String> fields) throws VariantAnnotationNotFoundException, VariantAnnotationWebServiceException;
+        String variant,
+        String isoformOverrideSource,
+        Map<String, String> token,
+        List<AnnotationType> annotationTypeList) throws VariantAnnotationNotFoundException, VariantAnnotationWebServiceException;
```

Comment:
I think we should keep the param name as is for consistency

```suggestion
        List<AnnotationType> fields) throws VariantAnnotationNotFoundException, VariantAnnotationWebServiceException;
```